SpatialGDK/Documentation/features.md

```diff
@@ -48,6 +48,10 @@ Actor property handover is a new feature we’re introducing with the SpatialOS
 
 `Singleton Actors` is a new term that describes Actors which contain global state and logic.  Only one of these Actors exists in a game world. The SpatialOS Unreal GDK supports both client accessible and server only singletons (respectively known as public and private singletons). For more information see the documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors).
 
+## Cross-server RPCs
+
+`Cross-server RPCs` is a new type of RPC that functions between two servers. These will enable a server without authority over an Actor to communicate with the server which has authority and is able to manipulate the Actor. For more information, see [Cross-server RPC]({{urlRoot}}/content/dummy-link) documentation.
+
 ## Networking switch
 
-We’ve implemented a way to switch between the native Unreal networking and SpatialOS networking via an editor tickbox. This can be found inside `Play` dropdown option in the toolbar under `Spatial Networking`. This is useful to test valid functionality on a single server, and performance. See the documentation on the networking switch on the [Troubleshooting]({{urlRoot}}/content/troubleshooting) page.
+We’ve implemented a convenient way to switch between the native Unreal networking and SpatialOS networking via an editor tickbox. This can be found inside `Play` dropdown option in the toolbar under `Spatial Networking`. This is useful to test valid functionality and performance on a single server. See the documentation on the networking switch on the [Troubleshooting]({{urlRoot}}/content/troubleshooting) page.
```

Comment:
We’ve implemented a convenient way to switch between the native Unreal networking and SpatialOS networking via an editor tickbox. This can be found inside `Play` dropdown option in the toolbar under `Spatial Networking`. This is useful to test valid functionality and performance on a single server. See the documentation on the networking switch on the [Troubleshooting]({{urlRoot}}/content/troubleshooting) page.

->(This needs tech writer-ing :-) )

The GDK offers you a convenient way to switch between native Unreal networking and SpatialOS networking via a checkbox in the Unreal Editor. This is useful to test valid functionality and performance on a single server. See the documentation on the networking switch on the [Troubleshooting]({{urlRoot}}/content/troubleshooting) page.
To switch networking to SpatialOS, from the Editor toolbar menu, select **Spatial Networking** > **Play** > **????**
[Editor note: is the "????" an option from "Play"? or a pair of options??)

Reply:
I've reworded your version - we want people to use this by default and only uncheck if something is wrong. Feel free to rephrase my version after I commit.